docs/tables/kubernetes_pod_disruption_budget.md

```diff
@@ -0,0 +1,37 @@
+# Table: kubernetes_pod_disruption_budget
+
+A Pod Disruption Budgeti (PDB) limits the number of Pods of a replicated application that are down simultaneously from voluntary disruptions.
+
+## Examples
+
+### Basic Info
+
+```sql
+select
+  name,
+  namespace,
+  min_available,
+  max_unavailable,
+  selector
+from
+  kubernetes_pod_disruption_budget
+order by
+  namespace, name;
+```
+
+### List Deployments and their matching PDB 
+
+```sql
+select
+  d.namespace,
+  d.name,
+  min_available,
+  replicas
+from
+  kubernetes_pod_disruption_budget pdb
+inner join
+  kubernetes_deployment d on d.selector = pdb.selector and d.namespace = pdb.namespace
+order by
+  d.namespace, d.name
```

Comment:
```suggestion
select
  d.namespace,
  d.name,
  min_available,
  replicas 
from
  kubernetes_pod_disruption_budget pdb 
  inner join
    kubernetes_deployment d 
    on d.selector = pdb.selector 
    and d.namespace = pdb.namespace 
order by
  d.namespace,
  d.name;
```

For our queries, we use a SQL formatter as given in https://steampipe.io/docs/develop/table-docs-standards#style-conventions-1 to help standardize formatting.